pkg/agent/openflow/client.go

```diff
@@ -374,6 +375,27 @@ type Client interface {
 	// or ip, port, protocol and direction. It is used to bypass NetworkPolicy enforcement on a VM for the particular
 	// traffic.
 	InstallPolicyBypassFlows(protocol binding.Protocol, ipNet *net.IPNet, port uint16, isIngress bool) error
+
+	// GetMeterStats sends a multipart request to get all the meter statistics and
+	// sets values for antrea_agent_ovs_meter_packet_count.
+	GetMeterStats() error
+}
+
+// GetMeterStats sends a multipart request to get all the meter statistics.
+func (c *client) GetMeterStats() error {
+	handleMeterStatsReply := func(packetCounts map[int]int64) {
+		for k, v := range packetCounts {
+			if k == PacketInMeterIDNP {
+				metrics.OVSMeterPacketCount.WithLabelValues("PacketInMeterNetworkPolicy").Set(float64(v))
+			} else if k == PacketInMeterIDTF {
+				metrics.OVSMeterPacketCount.WithLabelValues("PacketInMeterTraceflow").Set(float64(v))
+			} else {
+				klog.InfoS("Received unexpected meterId", "meterId", k)
+				continue
+			}
```

Comment:
I feel it should be clearer if you use `switch/case`

Reply:
Thanks, done.